gulp/tasks/webpack.js

```diff
@@ -78,7 +79,6 @@ function startWebpack(cb) {
       ]
     },
     externals: {
-      jquery: 'jQuery',
```

Comment:
Maybe the problem is related to this. Within a lot of the other javascript files, we currently assume that global jquery is available. With this change, it might be disrupting the other files besides the jquery one and how jquery is made available to them.